### PR TITLE
Fixed issue with Paths-integers being converted using local culture.

### PR DESCRIPTION
**Pull request by bergmania**
# Details
- Fixed issue with Paths-integers being converted using local culture.
- Fixed a  couple of issues with `AuthorizationHandler`s not getting multiple values from querystring correct
- Fixed a casing issue...

## Test
- Change user to swedish
- Try to create content in the root (not possible without this fix)
- Try show context menu on recycle bins (Not possible without this fix)

### CODE REVIEW

File: src/Umbraco.Core/Models/UserExtensions.cs [hunk 1]
```diff
@@ -219,14 +219,14 @@ private static bool StartsWithPath(string test, string path)
 
         private static string GetBinPath(UmbracoObjectTypes objectType)
         {
-            var binPath = Constants.System.Root + ",";
+            var binPath = Constants.System.RootString + ",";
             switch (objectType)
             {
                 case UmbracoObjectTypes.Document:
-                    binPath += Constants.System.RecycleBinContent;
+                    binPath += Constants.System.RecycleBinContentString;
```
Comment (p-m-j) [inline src/Umbraco.Core/Models/UserExtensions.cs:226]: This line could just be 
```
return Constants.System.RecycleBinContentPathPrefix; // "-1,-20"
```

File: src/Umbraco.Core/Models/UserExtensions.cs [hunk 2]
```diff
@@ -219,14 +219,14 @@ private static bool StartsWithPath(string test, string path)
...
                     break;
                 case UmbracoObjectTypes.Media:
-                    binPath += Constants.System.RecycleBinMedia;
+                    binPath += Constants.System.RecycleBinMediaString;
```
Comment (p-m-j) [inline src/Umbraco.Core/Models/UserExtensions.cs:229]: As above but RecycleBinMediaPathPrefix

File: src/Umbraco.Core/Services/UserServiceExtensions.cs [hunk 3]
```diff
@@ -12,9 +13,9 @@ public static class UserServiceExtensions
         public static EntityPermission GetPermissions(this IUserService userService, IUser user, string path)
         {
             var ids = path.Split(Constants.CharArrays.Comma, StringSplitOptions.RemoveEmptyEntries)
-                .Select(x => x.TryConvertTo<int>())
-                .Where(x => x.Success)
-                .Select(x => x.Result)
+                .Select(x => int.TryParse(x, NumberStyles.Integer, CultureInfo.InvariantCulture, out var value) ? (int?)value : null)
```
Comment (p-m-j) [inline src/Umbraco.Core/Services/UserServiceExtensions.cs:16]: Could be consistent with `Attempt<int>.Succeed/Fail`

File: src/Umbraco.Web.UI/umbraco/config/lang/da.xml [hunk 4]
```diff
@@ -18,7 +18,7 @@
     <key alias="delete">Slet</key>
     <key alias="disable">Deaktivér</key>
     <key alias="editSettings">Edit settings</key>
-    <key alias="emptyRecycleBin">Tøm papirkurv</key>
+    <key alias="emptyrecyclebin">Tøm papirkurv</key>
     <key alias="enable">Aktivér</key>
     <key alias="exportDocumentType">Eksportér dokumenttype</key>
     <key alias="importDocumentType">Importér dokumenttype</key>
```
Comment (p-m-j) [inline src/Umbraco.Web.UI/umbraco/config/lang/da.xml:24]: Are these not a problem? e.g. importdocumenttype.html vs importDocumentType

**Review by p-m-j — APPROVED** [review]

File: src/Umbraco.Core/Models/UserExtensions.cs (hunk 1 above)
Comment (bergmania) [reply src/Umbraco.Core/Models/UserExtensions.cs:226]: Constants.System.RecycleBinContentPathPrefix  has a "," in the end

File: src/Umbraco.Core/Models/UserExtensions.cs (hunk 2 above)
Comment (bergmania) [reply src/Umbraco.Core/Models/UserExtensions.cs:229]: also ending ","

**Pull request merged**